CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# v0.5.0 (November 2020)
+- Various general (but very important) speed improvements
```

Comment:
Just FYI, in the changelogs of most other projects of `aiidateam` we include a link to the relevant PR, e.g.:
- Add concept of (unique) `container_id` [[#97]](https://github.com/aiidateam/disk-objectstore/pull/97)
Up to you if you want to add it.

Reply:
Indeed, I didn't bother doing it here, but it's a good thing. It's added now (only for v0.5, for older versions it's not really worth it I think, they weren't production-ready).
If you or @chrisjsewell approve it and merge it, let me know please so I release it on pypi.